Blog/GamingGuruBlog.Web/GamingGuruBlog.Web/UIConverter.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using GamingGuruBlog.Domain;
+using GamingGuruBlog.Domain.Interfaces;
+using GamingGuruBlog.Domain.Models;
+using GamingGuruBlog.Web.Models;
+using System.Web.Mvc;
+
+namespace GamingGuruBlog.Web
+{
+    public class UIConverter
```

Comment:
@wojonet , I made these changes and commited them. 
Okay, I may have over done it but I'm not sure. When a BlogPostVM is turned to a BlogPost a lot items are created in the DB and then associated with that BlogPost. So, this is what is happening in the conversion and at the end it returns a BlogPost with all the associations. Is this correct?